testsuite/classlibrary/TestGraphBuilder.sc

```diff
@@ -0,0 +1,11 @@
+
+TestEnvGate : UnitTest {
```

Comment:
This file should be named TestEnvGate; if a more specific name is desired you could use TestEnvGate_GraphBuilding

Reply:
done, thanks!